W-14476576 DC sync not req duke (includes W-14476577,  W-14476579,  W-14776338) (#263)
mulesoft/docs-code-builder: modules/ROOT/pages/des-delete-api-projects.adoc

. Right-click and select to delete the folder.

[[delete-api-project]]
== Delete an API Design Project

// note on limited scope of this section:
include::partial$acb-api-design.adoc[tags="api-dc-scm-scope"]

> Design project names must be unique. To create a new design project with the same name in Anypoint Code Builder, you must delete the existing project from the IDE. If your design project is stored in the Anypoint SCM, it also exists in Design Center. You can delete also it from Design Center if you choose.

Reviewer: ```suggestion
Design project names must be unique. To create a new design project with the same name in Anypoint Code Builder, you must delete the existing project from the IDE. If your design project is stored in the Anypoint SCM, it also exists in Design Center. You can also delete it from Design Center if you choose.
```

[[delete-api-project]]
== Delete an API Project from Design Center

// note on limited scope of this section:
include::partial$acb-api-design.adoc[tags="api-dc-scm-scope"]

> Design project names must be unique. If your design project is stored in the Anypoint SCM, it also exists in Design Center. To create a new design project with the same name in Anypoint Code Builder, you must delete the existing project not only from Anypoint Code Builder but also from Design Center. If you want the project to remain in the Anypoint SCM, do not delete it from Design Center.

Reviewer 2: This is no longer required. ACB project creation does not create a Design Center project, so there is no limitation on project naming other than if there is already a folder created with the same name that the one that is being created as part of the project creation in the user's local machine.

Author: If the design project was imported or created before the Feb release, it will be stored in the Anypoint SCM. This doesn't apply to other design projects not stored there.

Reviewer 2: This part
> To create a new design project with the same name in Anypoint Code Builder, you must delete the existing project not only from Anypoint Code Builder but also from Design Center.

Is no longer true. Project creation won't conflict with design center project names anymore. Even when importing, you are reusing the already created project folder.

Author: ```suggestion
Design project names must be unique. If your design project is stored in the Anypoint SCM, it also exists in Design Center. In this case, when you create a new design project with the same name in Anypoint Code Builder, you must delete the existing project not only from Anypoint Code Builder but also from Design Center. If you want the project to remain in the Anypoint SCM, do not delete it from Design Center.
```

Author: ok, I see what you mean. Thx! I checked and was able to create a design project with the same name in the IDE after deleting a project that was imported from DC and connected to the SCM. There was no need to delete the project in DC anymore.

[[delete-api-project]]
== Delete an API Project from Design Center

Design project names must be unique. If your design project is stored in the Anypoint SCM, it also exists in Design Center. To create a new design project with the same name in Anypoint Code Builder, you must delete the existing project not only from Anypoint Code Builder but also from Design Center. If you want the project to remain in the Anypoint SCM, do not delete it from Design Center.

> This process applies only to API design projects imported to Anypoint Code Builder from Design Center, which automatically sync with the Anypoint SCM, and design projects created in Anypoint Code Builder before the February 2024 release. For more information about the Anypoint SCM, see start-scm.adoc[].

Author: ```suggestion
This process applies only to API design projects imported to Anypoint Code Builder from Design Center and design projects created in Anypoint Code Builder before the February 2024 release. Both sets of projects automatically sync with the Anypoint SCM. For more information about the Anypoint SCM, see start-scm.adoc[].
```

Author: make sure xref:start-scm.adoc exists
